src/Plugin.php

```diff
@@ -86,6 +90,45 @@ public static function upload_mime_types(array $mimes) {
   }
 
   /**
+   * Removes svg width and height attributes from the content.
+   *
+   * @implements the_content
+   */
+  public static function the_content($content) {
+    $content = preg_replace_callback('@<img [^>]*src=.+?.svg[^>]*>@', function ($matches) {
+      return preg_replace('@((?:width|height)=.+?\s+)@', '', $matches[0]);
```

Comment:
1. Doesn't make a practical difference in this case here, as far as I can see.

1. 👍

1. Can we fix the SCRM database (changing 128 into 1) and make this code look for the explicit case of `width="1" height="1"`?